services/QuillLMS/client/app/bundles/Shared/components/shared/tooltip.tsx

```diff
@@ -27,42 +28,43 @@ export class Tooltip extends React.Component<TooltipProps, {}> {
     this.hideTooltip = this.hideTooltip.bind(this)
   }
 
-
-  componentWillUnmount() {
-    document.removeEventListener('click', this.hideTooltipOnClick, false)
-  }
-
-  hideTooltipOnClick(e) {
-    const { handleClick } = this.props;
-    if (this.tooltip && e.target !== this.tooltip && e.target !== this.tooltipTrigger) {
-      this.hideTooltip()
-    }
-    if(handleClick) {
-      handleClick(e);
-    }
```

Comment:
I think there is behavior here that isn't really captured by the new setup - namely, that the tooltip is hidden when you click somewhere on the page that isn't the tooltip (behavior we probably want to keep), and also that if a `handleClick` prop is passed in we do call it when it's clicked. I'm not sure where the latter case is actually in use, so maybe it isn't, but we should confirm that before we remove it.

Reply:
I actually think the only place it's currently in use is from that `handleImageAttributionClick` in Evidence.